opal/util/info_subscriber.c

```diff
@@ -266,13 +265,10 @@ int opal_infosubscribe_change_info(opal_infosubscriber_t *object, opal_info_t *n
             key_str = iterator->ie_key;
             OBJ_RETAIN(key_str);
 
-            updated_value = opal_infosubscribe_inform_subscribers(object, iterator->ie_key->string,
-                                                                  iterator->ie_value->string,
-                                                                  &found_callback);
-            if (NULL != updated_value
-                && 0 != strncmp(updated_value, value_str->string, value_str->length)) {
-                err = opal_info_set(object->s_info, iterator->ie_key->string, updated_value);
-            }
+            opal_infosubscribe_inform_subscribers(object, iterator->ie_key->string,
```

Comment:
Reading `opal_infosubscribe_inform_subscribers` I do not think the return value has any significance. Hence we should always update `object->s_info`